src/Cli/ConfigGenerator.cs

```diff
@@ -949,9 +949,9 @@ public static bool VerifyCanUpdateRelationship(RuntimeConfig runtimeConfig, stri
         public static bool TryStartEngineWithOptions(StartOptions options)
         {
             string? configToBeUsed = options.Config;
-            if (string.IsNullOrEmpty(configToBeUsed))
+            if (string.IsNullOrEmpty(configToBeUsed) && TryMergeConfigsIfAvailable(out configToBeUsed))
             {
-                configToBeUsed = MergeConfigsIfAvailable();
+                _logger.LogError($"Using Merged Config File based on environment:{configToBeUsed}.");
```

Comment:
Why shouldn't this be an informational log?

Reply:
sorry, my bad. it should be informational